packages/console/src/app/referrals/page.tsx

```diff
@@ -134,8 +138,9 @@ export default function ReferralsPage () {
         ) : (
           <>
             <ReferralsList />
-            {referralLink ? (
-              <RefcodeLink referralLink={referralLink} />
+            
+            {referralLink ? ( 
+                <RefcodeLink referralLink={referralLink} />
```

Comment:
formatting change not needed

```suggestion
              <RefcodeLink referralLink={referralLink} />
```